docs/reference/synonyms/apis/put-synonyms-set.asciidoc

```diff
@@ -0,0 +1,215 @@
+[[put-synonyms-set]]
+=== Put synonyms set
+
+beta::[]
+
+++++
+<titleabbrev>Put synonyms set</titleabbrev>
+++++
+
+Creates or updates a synonyms set.
+
+[[put-synonyms-set-request]]
+==== {api-request-title}
+
+`PUT _synonyms/<synonyms_set>`
+
+[[put-synonyms-set-prereqs]]
+==== {api-prereq-title}
+
+Requires the `manage_search_synonyms` cluster privilege.
+
+[[put-synonyms-set-path-params]]
+==== {api-path-parms-title}
```

Comment:
I think it would be better to have a section for query parameters and a section for the request body here. See https://www.elastic.co/guide/en/elasticsearch/reference/master/indices-put-template.html#put-index-template-api-path-params for example.

Reply:
Thanks for the link, changing it!